packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoiceDiscount/en.ftl

```diff
@@ -21,6 +21,13 @@ subscriptionFirstInvoiceDiscount-content-subtotal = Subtotal: { $invoiceSubtotal
 # Variables:
 #  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
 subscriptionFirstInvoiceDiscount-content-discount = Discount: -{ $invoiceDiscountAmount }
+# Variables
+#  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
+subscriptionFirstInvoiceDiscount-content-discount-one-time = One Time Discount: -{ $invoiceDiscountAmount }
```

Comment:
Small one, but could be difficult to fix later, in the Jira ticket it says the texts should  be `One time Discount` and `x month Discount`.

Reply:
done